cryptography-core/src/dalek_wrapper/mod.rs

```diff
@@ -0,0 +1,80 @@
+use super::cdd_claim::{RISTRETTO_POINT_SIZE, SCALAR_SIZE}; // todo: maybe move these here.
+use codec::{Decode, Encode, Error as CodecError, Input, Output};
+use curve25519_dalek::{
+    ristretto::{CompressedRistretto, RistrettoPoint},
+    scalar::Scalar,
```

Comment:
change these to `scalar::Scalar as DalekScalar` and then, you can give the new types Scalar name. E.g., `struct Scalar(pub DalekScalar)`. Similarly for RistrettoPoint